internal/provider/resource_deployment.go

```diff
@@ -26,233 +25,244 @@ func resourceDeployment() *schema.Resource {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "List of disk workloads configurations.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": {
 							Type:        schema.TypeString,
 							Required:    true,
-							Description: "the disk name, used to reference it in zmachine mounts",
+							Description: "Disk workload name.",
 						},
 						"size": {
 							Type:        schema.TypeInt,
 							Required:    true,
-							Description: "the disk size in GBs",
+							Description: "Disk size in GBs.",
 						},
 						"description": {
-							Type:     schema.TypeString,
-							Optional: true,
-							Default:  "",
+							Type:        schema.TypeString,
+							Optional:    true,
+							Default:     "",
+							Description: "Description for disk workload.",
 						},
 					},
 				},
 			},
 			"zdbs": {
-				Type:     schema.TypeList,
-				Optional: true,
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "List of Zdb workloads configurations. You can read more about 0-db (Zdb) [here](https://github.com/threefoldtech/0-db/).",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": {
-							Type:     schema.TypeString,
-							Required: true,
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Zdb worklod name.",
 						},
 						"password": {
-							Type:     schema.TypeString,
-							Required: true,
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Zdb password.",
 						},
 						"public": {
 							Type:        schema.TypeBool,
 							Optional:    true,
 							Default:     false,
-							Description: "Makes it read-only if password is set, writable if no password set",
+							Description: "Makes it read-only if password is set, writable if no password set.",
 						},
 						"size": {
 							Type:        schema.TypeInt,
 							Required:    true,
-							Description: "Size of the zdb in GBs",
+							Description: "Size of the zdb in GBs.",
 						},
 						"description": {
-							Type:     schema.TypeString,
-							Optional: true,
-							Default:  "",
+							Type:        schema.TypeString,
+							Optional:    true,
+							Default:     "",
+							Description: "Zdb workload description.",
 						},
 						"mode": {
 							Type:        schema.TypeString,
 							Optional:    true,
 							Computed:    true,
-							Description: "Mode of the zdb, user or seq",
+							Description: "Mode of the zdb, `user` or `seq`. `user` is the default mode where a user can SET their own keys, like any key-value store. All keys are kept in memory. in `seq` mode, keys are sequential and autoincremented.",
 						},
 						"ips": {
 							Type: schema.TypeList,
 							Elem: &schema.Schema{
 								Type: schema.TypeString,
 							},
 							Computed:    true,
-							Description: "IPs of the zdb",
+							Description: "Computed IPs of the zdb.",
```

Comment:
Are these IPv6 or wireguard?

Reply:
Two ips are computed: a public ipv6, and a ygg ip, in this order.
will add this also.